app/views/site/welcome.html.erb

```diff
@@ -6,66 +6,66 @@
   <h1><%= t ".title" %></h1>
 <% end %>
 
-<p><%= t ".introduction_html" %></p>
+<p class="lead"><%= t ".introduction_html" %></p>
 
-<h3><%= t ".whats_on_the_map.title" %></h3>
+<h2><%= t ".whats_on_the_map.title" %></h2>
 
 <div class='row'>
   <div class='col'>
-    <div class='center'>
-      <span class='sprite small check'></span>
+    <div>
+      <span class='sprite small check mx-auto'></span>
     </div>
     <p><%= t ".whats_on_the_map.on_html" %></p>
   </div>
   <div class='col'>
     <div class='center'>
-      <span class='sprite small x'></span>
+      <span class='sprite small x mx-auto'></span>
     </div>
     <p><%= t ".whats_on_the_map.off_html" %></p>
   </div>
 </div>
 
-<h3><%= t ".basic_terms.title" %></h3>
+<h2><%= t ".basic_terms.title" %></h2>
 
 <p><%= t ".basic_terms.paragraph_1_html" %></p>
 
-<div class='clearfix icon-list'>
+<div class='clearfix'>
   <div class='clearfix'>
-    <span class='sprite small term editor'></span>
-    <p><%= t ".basic_terms.editor_html" %></p>
+    <span class='sprite small term editor float-start'></span>
+    <p class="align-middle"><%= t ".basic_terms.editor_html" %></p>
```

Comment:
This `align-middle` is not working because it's on a block element - it's also missing from the other three entries below.

Reply:
oops, I had been doing some experiments, and that wasn't intended for this PR.

Reply:
fixed